static/src/javascripts/vendor/omniture.js

```diff
@@ -69,13 +69,14 @@ function s_doPlugins(s) {
     for (var i=0,j=diary.length;i<j;++i)
     {
         var pV=diary[i];
-        var tmp=(pV>=cutoff) && (pV!=now) && (vC+=1) && newDiary.push(pV);
+        var tmp=(pV>=cutoff) && (pV<now) && (vC+=1) && newDiary.push(pV);
```

Comment:
Yes, it appears there were issues where we were receiving dates in the future.. (?) so this fixes that.